models.py

```diff
@@ -17,6 +17,7 @@ class Node(db.Model):
     connection_config = db.Column(db.Text)
     created_at = db.Column(db.DateTime)
     updated_at = db.Column(db.DateTime)
+    # TODO: add signature field
```

Comment:
What signature is this referring to?

Reply:
removed todo, out of scope of this branch